<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillStats</class>
 <widget class="QWidget" name="SkillStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutMain">
   <item>
    <layout class="QHBoxLayout" name="hLayoutMenu"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutClassEngrave">
     <item>
      <widget class="QLabel" name="labelInfo">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutSkill"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
